source/_integrations/statistics.markdown

```diff
@@ -30,11 +30,11 @@ The `statistics` integration is different to [Long-term Statistics](https://deve
 
 ## Characteristics
 
-The following statistical characteristics are available. Pay close attention to the right configuration of `sampling_size` and/or `max_age`, as most characteristics are directly influenced by these settings.
+The following statistical characteristics (`state_characteristic`) are available. Pay close attention to the right configuration of `sampling_size` and/or `max_age`, as most characteristics are directly influenced by these settings.
 
 ### Numeric Source Sensor
 
-The following characteristics are supported for `sensor` source sensors:
+The following are supported for `sensor` source sensors `state_characteristic`:
```

Comment:
I think this one flows worse (same for the binary further below).   Can the word "source" be eliminated?  I know the following isn't perfect, but maybe something like:
```
The following `state_characteristic`s are supported by the `sensor` type.
```

Reply:
This documentation document talks about the "source sensor" or "source entity" consistently, to establish a clear difference between the created statistics sensor, and the underlying source sensor it is derived from.

```suggestion
The following options for `state_characteristic` are supported for source entities of the `sensor` domain:
```